fluentd/configs.d/openshift/output-operations.conf

```diff
@@ -1,6 +1,6 @@
 <match journal.system** system.var.log** **_default_** **_openshift_** **_openshift-infra_** mux.ops>
   @type copy
-  @include output-es-ops-config.conf
+  @include ../dynamic/output-es-ops-config.conf
```

Comment:
No, this is for this PR to manage the configuration files.

If, e.g., output-es-ops-config.conf is to be configured, this copy is executed:
`cp $CFG_DIR/{openshift,dynamic}/output-es-ops-config.conf`
otherwise the dynamic/output-es-ops-config.conf would be empty and copy won't happen.
`echo > $CFG_DIR/dynamic/output-es-ops-config.conf`